docs/framework/react/reference/useMutation.md

```diff
@@ -94,7 +94,7 @@ mutate(variables, {
   - Defaults to a unique id (so that all mutations run in parallel)
   - Mutations with the same scope id will run in serial
 - `throwOnError: undefined | boolean | (error: TError) => boolean`
-  - Defaults to the global query config's `throwOnError` value, which is `undefined`
+  - Defaults to the global `QueryClient`'s `defaultOptions.queries.throwOnError`.
```

Comment:
Is this true for _mutations_? Ref: https://github.com/TanStack/query/issues/8569